index.html

```diff
@@ -1705,29 +1705,36 @@ <h3>Validity Period</h3>
       </section>
 
       <section>
-        <h3>Proofs (Signatures)</h3>
+        <h3>Securing Verifiable Credentials</h3>
 
         <p>
-At least one proof mechanism, and the details necessary to evaluate that proof,
-MUST be expressed for a <a>credential</a> or <a>presentation</a> to be a
+At least one securing mechanism, and the details necessary to evaluate it, MUST
+be expressed for a <a>credential</a> or <a>presentation</a> to be a
 <a>verifiable credential</a> or <a>verifiable presentation</a>; that is, to be
 <a>verifiable</a>.
         </p>
-
         <p>
-This specification identifies two classes of proof mechanisms: external proofs
-and embedded proofs. An <dfn class="lint-ignore">external proof</dfn> is one
-that wraps an expression of this data model, such as a JSON Web Token, which is
-elaborated on in the Securing Verifiable Credentials using JSON Web Tokens
-[[?VC-JWT]] specification. An <dfn>embedded proof</dfn> is a mechanism where
-the proof is included in the data, such as a Data Integrity Proof, which is
-elaborated upon in Verifiable Credential Data Integrity [[VC-DATA-INTEGRITY]].
+This specification recognizes two classes of securing mechanisms: those that use
+external proofs and those that use embedded proofs. An
+<dfn class="lint-ignore">external proof</dfn> is one that wraps an expression of
+this data model, such as a JSON Web Token, which is elaborated on in the
+Securing Verifiable Credentials using JSON Web Tokens [[VC-JWT]] specification.
+An <dfn>embedded proof</dfn> is a mechanism where the proof is included in the
+data model, such as a Data Integrity Proof, which is elaborated on in Verifiable
+Credential Data Integrity [[VC-DATA-INTEGRITY]].
```

Comment:
It's important to make external document titles distinct from the body text which includes them.
```suggestion
<em>Securing Verifiable Credentials using JSON Web Tokens</em> [[VC-JWT]] specification.
An <dfn>embedded proof</dfn> is a mechanism where the proof is included in the
data model, such as a Data Integrity Proof, which is elaborated on in <em>Verifiable
Credential Data Integrity</em> [[VC-DATA-INTEGRITY]].
```